gnosis/safe/safe.py

```diff
@@ -431,32 +431,40 @@ def estimate_tx_gas_with_web3(self, to: str, value: int, data: bytes) -> int:
         """
         Estimate tx gas using web3
         """
-        return self.ethereum_client.estimate_gas(self.address, to, value, data)
+        gas_estimated = self.ethereum_client.estimate_gas(self.address, to, value, data)
+        for _ in range(20):  # Test `call()` and increase gas 20 times, 63/64th problem
+            try:
```

Comment:
This will not work for delegate calls right?

Reply:
No, delegate calls are disabled for his kind of estimation: https://github.com/gnosis/gnosis-py/blob/eip-150-fix/gnosis/safe/safe.py#L460

Reply:
it this does not throw then we just perform the same call 20 times?

Reply:
@rmeissner You are right, I fixed it